plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-checkout-restricted-coupons.spec.js

```diff
@@ -661,14 +661,16 @@ test.describe( 'Cart & Checkout Restricted Coupons', () => {
 		await page
 			.getByLabel( 'Email address' )
 			.first()
-			.fill( 'marge@example.com' );
+			.fill( 'marge.simpson@example.org' );
 
 		await page
 			.getByRole( 'link', { name: 'Click here to enter your code' } )
 			.click();
 		await page.getByPlaceholder( 'Coupon code' ).fill( 'email-restricted' );
 		await page.getByRole( 'button', { name: 'Apply coupon' } ).click();
 
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
Is waiting for network really necessary? The next expect should wait and timeout.

Reply:
Weirdly, without this in here explicitly, it would occasionally not wait and just report that the next element isn't visible.